src/pages/Encounters/EncounterShow.tsx

```diff
@@ -103,9 +103,8 @@ export const EncounterShow = (props: Props) => {
     }`;
 
   return (
-    <div>
-      <nav className="relative flex flex-wrap items-start justify-between">
-        <PageTitle title={t("encounter")} />
+    <Page title={t("encounter")}>
```

Comment:
Check on smaller form factors (344x882 for example).

Reply:
<img width="384" alt="image" src="https://github.com/user-attachments/assets/f07a405d-7bbb-4c44-8be7-65b58b94c297" />

Works fine IG